src/cpp/fastdds/topic/DDSSQLFilter/DDSFilterGrammar.hpp

```diff
@@ -57,6 +57,7 @@ struct integer_value : seq< opt< sign >, integer > {};
 struct fractional : seq< dot_op, integer > {};
 struct exponent : seq< one< 'e', 'E' >, integer_value > {};
 struct float_value : seq < opt< sign >, integer, sor < exponent, seq< fractional, opt< exponent > > > > {};
+struct hex_value : seq< opt< sign >, one< '0' >, one< 'x', 'X' >, plus< xdigit > > {};
```

Comment:
I've made a test, changing this `xdigit` by `digit`, simulating a developer error, and the new tests continue passing.

Reply:
Done in eb8b5dc1d00628b116e86b0d239d711c703bdd2b